infra/gcp/terraform/k8s-infra-prow-build/serviceaccounts.tf

```diff
@@ -26,16 +26,38 @@ locals {
       description       = "sync K8s secrets from GSM in this and other projects"
       project_roles     = ["roles/secretmanager.secretAccessor"],
       cluster_namespace = "kubernetes-external-secrets"
+      additional_workload_identity_principals = [
+        "principalSet://iam.googleapis.com/${google_iam_workload_identity_pool_provider.eks_cluster.name}/attribute.sub/system:serviceaccount:default:default"
```

Comment:
To what `default` SA is this referring to? Is it SA in GKE or EKS cluster?

Reply:
Updated it to refer to the correct EKS pod SA. `system:serviceaccount:external-secrets:external-secrets`

/label tide/merge-method-squash